src/LazyLoad.jsx

```diff
@@ -35,7 +35,11 @@ export default class LazyLoad extends Component {
 
     if (top === 0 || (top <= (scrollTop + window.innerHeight + threshold)
                       && (top + height) > (scrollTop - threshold))) {
-      this.setState({ visible: true });
+      this.setState({ visible: true }, () => {
+        if (onContentVisible) {
```

Comment:
I'm relatively new to React and have not heard of using the callback of setState as being a bad idea. Could you give a bit of insight into why this is such?